Assets/Content/Items/Consumables/Food/Consumable.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Mirror;
+using SS3D.Content.Systems.Interactions;
+using SS3D.Engine.Interactions;
+using SS3D.Engine.Interactions.Extensions;
+using SS3D.Engine.Inventory;
+using SS3D.Engine.Inventory.Extensions;
+using SS3D.Engine.Substances;
+using UnityEngine;
+
+
+/// <summary>
+/// This class handles items that can be consumed
+/// they have an substance container which has have it's contents processed when ingested
+/// </summary>
+public class Consumable : Item, IConsumable
```

Comment:
If you merge master you can use IInteractionSourceExtension and won't have to inherit from the item class.
That makes it easier to add to existing items.

Reply:
Didn't used this you fix that later lol